Categories/Categories.html

```diff
@@ -40,7 +40,7 @@ <h2 class="header-title">
                     <p class="text-sm card-des">Covid Essentials</h4>
```

Comment:
You html and css file name are starting from capital letters.
It is recommended to use only small case characters for html and css files

Reply:
I suppose this will not matter as we are going to convert these  pages into reusable components.